### DISCUSSION TITLE
error with devtools:install()

### DISCUSSION THREAD

**Discussion by HettiePike** [IBAHCM/RPiR#340] [Archive] [answered]
Hello @richardreeve , @soniamitchell 

I'm getting this error message every time I do devtools:instll()
Error: HTTP error 403.
  API rate limit exceeded for 5.66.176.168. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)

  Rate limit remaining: 0/60
  Rate limit reset at: 2021-02-26 21:49:17 UTC

  To increase your GitHub API rate limit
  - Use `usethis::browse_github_pat()` to create a Personal Access Token.
  - Use `usethis::edit_r_environ()` and add the token as `GITHUB_PAT`.

I've tried running the two things they suggest but I just get back more errors.
Thanks

**Comment by soniamitchell** [comment, marked as answer]
Hi @HettiePike try this https://ibahcm.github.io/RPiR/articles/pages/install_github.html#add-a-pat 👍

**Comment by HettiePike** [comment]
Thanks so much, Sonia!!
